src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestCollection.cs

```diff
@@ -97,6 +98,14 @@ void DumpCounter(QUIC_PERFORMANCE_COUNTERS counter)
         Console.WriteLine($"Unobserved exceptions of {s_unobservedExceptions.Count} different types: {Environment.NewLine}{string.Join(Environment.NewLine + new string('=', 120) + Environment.NewLine, s_unobservedExceptions.Select(pair => $"Count {pair.Value}: {pair.Key}"))}");
     }
 
+    internal static bool IsWindowsVersionWithSchannelSupport()
+    {
+        // copied from MsQuicApi implementation to avoid triggering the static constructor
+        Version minWindowsVersion = new Version(10, 0, 20145, 1000);
+        return OperatingSystem.IsWindowsVersionAtLeast(minWindowsVersion.Major,
+        minWindowsVersion.Minor, minWindowsVersion.Build, minWindowsVersion.Revision);
```

Comment:
NIT:
no new-line
```suggestion
        return OperatingSystem.IsWindowsVersionAtLeast(minWindowsVersion.Major, minWindowsVersion.Minor, minWindowsVersion.Build, minWindowsVersion.Revision);
```
or with tab:
```suggestion
        return OperatingSystem.IsWindowsVersionAtLeast(minWindowsVersion.Major,
            minWindowsVersion.Minor, minWindowsVersion.Build, minWindowsVersion.Revision);
```